all.json

```diff
@@ -11,6 +11,82 @@
 		"162.222.226.37",
 		"1776coin.org",
 		"1chaincrypto.com",
+		"24x7-bittreks.com",
+		"24x7-ledger.com",
+		"alienheroesworld.io",
+		"amodosecurityservices.org",
+		"app-onlinelottoclaim.us",
+		"app.anchorappweb.com",
+		"app.cosmoioapp.com",
```

Comment:
Appears twice - `app` & `web` with no top-level blocks.

Reply:
added the top levels. 
they were already there
removed subdomains and redundant TLDs